src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_index_pattern/step_index_pattern.html

```diff
@@ -1,169 +1,170 @@
-            indices="stepIndexPattern.allIndices"
-            pattern="stepIndexPattern.indexPatternName"
-          >
-            <span class="kuiStatusText kuiStatusText--info">
-              <span>
-                The index pattern you've entered doesn't match any indices. You can match any of your <strong>{{stepIndexPattern.allIndices.length}} indices</strong>, below.
-              </span>
-            </span>
-          </matching-indices-list>
-        </div>
+  <!-- List of matching indices -->
+  <div ng-switch="stepIndexPattern.matchingIndicesListType">
+    <div ng-switch-when="invalidIndexPattern">
+      <matching-indices-list
+        is-loading="stepIndexPattern.isFetchingMatchingIndices"
+        indices="stepIndexPattern.allIndices"
+        pattern="stepIndexPattern.indexPatternName"
+      >
+        <span class="euiTextColor euiTextColor--danger">
+          <span aria-hidden="true">
+            <svg class="euiIcon euiIcon--medium" xmlns="http://www.w3.org/2000/svg"
+              width="16" height="16" viewBox="0 0 16 16">
+              <g fill="#13252D" fill-rule="evenodd">
+                <circle cx="5" cy="5" r="1" />
+                <circle cx="10" cy="5" r="1" />
+                <path fill-rule="nonzero" d="M7.5 14a6.5 6.5 0 1 0 0-13 6.5 6.5 0 0 0 0 13zm0 1a7.5 7.5 0 1 1 0-15 7.5 7.5 0 0 1 0 15z"
+                />
+                <path fill-rule="nonzero" d="M12.332 9.626C10.747 8.217 9.133 7.5 7.5 7.5s-3.247.717-4.832 2.126a.5.5 0 1 0 .664.748C4.747 9.116 6.133 8.5 7.5 8.5s2.753.616 4.168 1.874a.5.5 0 0 0 .664-.748z"
+                />
+              </g>
+            </svg>
+          </span>
 
-        <div ng-switch-when="partialMatches">
-          <matching-indices-list
-            is-loading="stepIndexPattern.isFetchingMatchingIndices"
-            indices="stepIndexPattern.partialMatchingIndices"
-            pattern="stepIndexPattern.indexPatternName"
-          >
-            <span class="kuiStatusText kuiStatusText--info">
-              <span>
-                Your index pattern doesn't match any indices, but you have <strong>{{stepIndexPattern.partialMatchingIndices.length}} {{stepIndexPattern.partialMatchingIndices.length > 1 ? 'indices' : 'index'}}</strong> which {{stepIndexPattern.partialMatchingIndices.length > 1 ? 'look' : 'looks'}} similar.
-              </span>
-            </span>
-          </matching-indices-list>
-        </div>
+          <span>
+            You've entered an invalid index pattern. Please adjust it to match any of your <strong>{{stepIndexPattern.allIndices.length}} indices</strong>, below.
+          </span>
+        </span>
+      </matching-indices-list>
+    </div>
+    <div ng-switch-when="noInput">
+      <matching-indices-list
+        is-loading="stepIndexPattern.isFetchingMatchingIndices"
+        indices="stepIndexPattern.allIndices"
+        pattern="stepIndexPattern.indexPatternName"
+      >
+        <span class="euiTextColor euiTextColor--primary">
```

Comment:
We're going to remove `euiTextColor--primary` from EUI because it looks too much like a link. So I think we can remove this wrapping `span` entirely.

Reply:
We need to remove references to `euiTextColor--primary` because it's going to be removed from EUI.